src/WordPress.php

```diff
@@ -237,9 +237,54 @@ public function create( $args, $assoc_args ) {
 		\EE::log( 'Configuring project.' );
 
 		$this->create_site( $assoc_args );
+
 		\EE\Utils\delem_log( 'site create end' );
 	}
 
+	/**
+	 * Enable redis object cache on wordpress site.
+	 */
+	private function enable_redis_cache() {
+
+		$activate_wp_redis_plugin = "docker-compose exec --user='www-data' php wp plugin install wp-redis --activate";
+		$enable_redis_cache       = "docker-compose exec --user='www-data' php wp redis enable";
+		$clone_nginx_helper       = 'docker-compose exec --user=\'www-data\' php git clone https://github.com/rtCamp/nginx-helper.git';
+		$activate_nginx_helper    = 'docker-compose exec --user=\'www-data\' php wp plugin activate nginx-helper';
+		$nginx_helper_fail_msg    = 'Unable to activate nginx-helper plugin';
+
+		$this->execute_system_command( $activate_wp_redis_plugin, 'Unable to download and activate wp-redis plugin.' );
+		$this->execute_system_command( $enable_redis_cache, 'Unable to enable object cache' );
+		$this->execute_system_command( $clone_nginx_helper, 'Unable to clone nginx-helper plugin' );
+
+		if ( \EE::exec( $activate_nginx_helper ) ) {
+			\EE::warning( $nginx_helper_fail_msg );
+		} else {
+			$add_hostname_constant = "docker-compose exec --user='www-data' php wp config set RT_WP_NGINX_HELPER_REDIS_HOSTNAME ee-global-redis --add=true --type=constant";
+			$add_port_constant     = "docker-compose exec --user='www-data' php wp config set RT_WP_NGINX_HELPER_REDIS_PORT 6379 --add=true --type=constant";
+			$add_prefix_constant   = "docker-compose exec --user='www-data' php wp config set RT_WP_NGINX_HELPER_REDIS_PREFIX nginx-cache: --add=true --type=constant";
+
+			$this->execute_system_command( $add_hostname_constant, $nginx_helper_fail_msg );
+			$this->execute_system_command( $add_port_constant, $nginx_helper_fail_msg );
+			$this->execute_system_command( $add_prefix_constant, $nginx_helper_fail_msg );
+
+		}
+	}
+
+	/**
+	 *  Execute command with fail msg.
+	 *
+	 * @param string $command  Command to execute.
+	 * @param string $fail_msg failure message.
+	 */
+	private function execute_system_command( $command, $fail_msg = '' ) {
```

Comment:
Don't name this `execute_system_command()` make it something like `docker_compose_exec()`. Create a `docker-compose exec` wrapper in [class-ee-docker.php](https://github.com/EasyEngine/easyengine/blob/79bf055857365f59f4ba2626f8c0a48d404d5a0f/php/class-ee-docker.php) . You can add parameters for selecting container, add `--user` if a user parameter is passed and a shell wrapper like `sh -c` or `bash -c` if a shell wrapper to execute commad is passed, even this can be empty by default.

Reply:
Added separate issue for tracking this issue.
https://github.com/EasyEngine/easyengine/issues/1227

Reply:
Yes, this would be better if tracked in a sperate issue as changes across other packages will also need an update after this.